lib/use-google-map.js

```diff
@@ -0,0 +1,170 @@
+// https://janosh.io/blog/google-maps+react-hooks, and own tweaks
+// input: address (string), updateAddress (function), updateVenueName (function)
+// output:
+//   - googleMap: contains the map component. use {googleMaps} to embed
+//   - results (array of objects): a list of auto-complete suggestions based on the address entered
+//   - setFocus(): called when the Search field is clicked on to trigger showing the results dropdown
+//   - setLocation(data): called when a location is selected via the results dropdown, triggers the visual map changes
+
+import React, { useState, useEffect, useRef, useCallback } from 'react';
+import { useScript } from './use-script';
+
+export const useGoogleMap = ({
```

Comment:
Since this is being used on the `host` app as well, I'm thinking I should add this either to some sort of universe utilities repo, or somewhere else... 🤔